src/components/EntityInfo.vue

```diff
@@ -108,14 +108,28 @@
 
         <v-col cols="12" md="3">
           <dl>
+
+            <!-- Registration Date -->
+            <template v-if="businessId && isFirm">
+              <dt class="mr-2">Registration Date:</dt>
+              <dd id="entity-business-registration-date">
+                {{ this.dateToPacificDate(getEntityFoundingDate) || 'Not Available' }}</dd>
+            </template>
+
+            <!-- Registration Number -->$
+            <template v-if="businessId && isFirm">
```

Comment:
Is the design to show both Registration Number and Business Number?

I _think_ those might be the same number with a different name. If so then you should not rendered both (ie, add condition to line 126).

Reply:
The design includes both but I will confirm whether the numbers represent the same thing or not

Reply:
Jyoti confirmed that both the Registration Number and Business Number should be displayed